### ISSUE TITLE
Update powertoys docs

### ISSUE THREAD

**Issue by VlxtIykg** [MicrosoftDocs/windows-uwp#4480]
Timezone plugin removed which I thought was really cool but I assumed it was removed due to its complications and it serves no purpose on the page anymore and might confuse new users such as myself!
![image](https://github.com/MicrosoftDocs/windows-dev-docs/assets/97310758/bebb92e7-ea81-4c5b-8072-669c36859e39)
![image](https://github.com/MicrosoftDocs/windows-dev-docs/assets/97310758/37ab41ae-5db9-4613-aebb-22f3a3cc4c35)

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0b951899-69ca-9057-f799-14491e1b0a74
* Version Independent ID: 18e0202f-3e91-a5b9-62f2-6cd40e5661a8
* Content: [PowerToys Run utility for Windows](https://learn.microsoft.com/en-us/windows/powertoys/run#features)
* Content Source: [hub/powertoys/run.md](https://github.com/MicrosoftDocs/windows-dev-docs/blob/docs/hub/powertoys/run.md)
* Product: **dev-environment**
* Technology: **powertoys**
* GitHub Login: @mattwojo
* Microsoft Alias: **mattwoj**

**Comment by mattwojo** [comment]
Hey @VlxtIykg - Thanks for filing this issue. Could you add a bit more clarification about what is confusing and what you think may help resolve this issue? Are you saying that you think "Removed the TimeZone plugin." should be removed from the Release Notes?

**Comment by VlxtIykg** [comment]
No no, not that! I believe the TimeZone plugin was removed from powertoys but it still showed as a possible plugin in the wiki and I was looking for ways to activate the plugin. After a little digging I found out that it was removed from powertoys in the release notes. It would save others the trouble if it was removed from the docs! https://learn.microsoft.com/en-us/windows/powertoys/run#features

TLDR: Remove time zones from https://learn.microsoft.com/en-us/windows/powertoys/run#direct-activation-commands in the list

**Comment by mattwojo** [comment]
Ahhh! Ha.. I had done a find on that page, but entered TimeZone as one word. I see it now -- thanks for clarifying! I'll remove that now. Thanks for pointing this out!!

**Comment by VlxtIykg** [comment]
No problem! Nice projects that are upcoming, all the best!! I'll close this issue now!
